src/panel/direction/RoadMapItem.jsx

```diff
@@ -7,7 +7,7 @@ const RoadMapItem = ({ children, icon, distance, className, line, ...rest }) =>
   <>
     <div className={classnames('itinerary_roadmap_item', className)} {...rest}>
       {line}
-      <Flex alignItems="center">
+      <Flex alignItems="flex-start">
```

Comment:
This component is also used in the roadmaps displayed as list, so it changes the alignement of the icon in it too.